### ISSUE TITLE
how is repo used in `expect_secure()`?

### ISSUE THREAD

**Issue by JosiahParry** [sonatype-nexus-community/oysteR#49]
@csgillespie, I'm currently trying to extend the testthat compatibility / functionality and am working off of your existing function. One thing that is not clear is the utility of the `repo` argument. Other than setting the repo option manually, I cannot figure out what it is used for. Based on my reading it is an unused argument. Can you clairfy?

https://github.com/sonatype-nexus-community/oysteR/blob/0a776638976d9f18e65ad4d17b806cfcceffeef7/R/expect_secure.R#L23

**Comment by csgillespie** [comment]
Yep this does seem like an odd argument. I think I came across an edge case where {testthat} stripped out the repos

https://github.com/sonatype-nexus-community/oysteR/blob/0a776638976d9f18e65ad4d17b806cfcceffeef7/R/expect_secure.R#L25

**Comment by JosiahParry** [comment]
Interesting. Perhaps it's my naiveté wrt {testthat}. Does testthat require that option be specified? Within `audit_description()` I don't think any function requires a repo to be specified. Safe to say I can continue developing omitting this arg unless any issue arises?

**Comment by csgillespie** [comment]
On Thu, 31 Dec 2020, 17:00 Josiah Parry, <notifications@github.com> wrote:

> Interesting. Perhaps it's my naiveté wrt {testthat}. Does testthat require
> that option be specified? Within audit_description() I don't think any
> function requires a repo to be specified.
>
The check package function calculates the dependency tree of packages. So
not just those in the description file. This requires a repo to be
specified


Safe to say I can continue developing omitting this arg unless any issue
> arises?
>
It should be obvious if it breaks.

My thinking is that sometimes you would specify a specific repo, eg if you
are using a tagged rspm.

I should probably document this😁
